mod/assets/set/abstract/asset_inputter.rb

```diff
@@ -33,7 +33,7 @@ def update_asset_input
   # otherwise the migration that adds the asset_input card fails
 
   Card::Auth.as_bot do
-    asset_input_card.update content: render_asset_input_content
```

Comment:
This was leading to double rendering, because the (virtual) asset input card was not using this content but rather its #virtual_content method (which also calls render_asset_input_content on this card).